### PR TITLE
feat: add article deepgram-speech-to-text-aws-lambda-websockets

### PR DESCRIPTION
**Pull request by naomi-lgbt**

### CODE REVIEW

File: articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - Ensure your Lambda function has permissions to access the Internet and the Deepgram API.
+
+4. **Deploy and Test**:
+    - Deploy the setup and perform end-to-end testing with different audio scenarios.
+
+### Conclusion
+
+Embedding Deepgram's transcription services within AWS serverless architecture streamlines the transcription process, enhances scalability, and significantly reduces the entry barrier for quick experimentation or deployment. Refer to the examples on Deepgram’s GitHub repositories for Node.js, Python, Go, and .NET for initial setups and adapt them as needed for AWS Lambda.
+
+### References
```
Comment (jpvajda) [inline articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md:75]: I'd expect to see some AWS references here.

File: articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Deepgram provides robust APIs for both pre-recorded and live transcription, ideal for real-time audio processing. Integrating these capabilities into AWS's serverless infrastructure allows for scalability and efficiency, reducing overhead costs and improving responsiveness.
+
+### Prerequisites
+
+- AWS Account with access to Lambda, API Gateway, and IAM roles.
+- Deepgram API Key.
+- Basic understanding of AWS Lambda and WebSockets.
+- Familiarity with any programming language supported by Deepgram SDKs: Node.js, Python, C#, Go, or Rust.
+
+### Setup Deepgram SDK
```
Comment (jpvajda) [inline articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md:18]: Remove code sample.

**Review by jpvajda — CHANGES_REQUESTED** [review]
see comments for suggestions.

File: articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md (hunk 1 above)
Comment (naomi-lgbt) [reply articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md:75]: https://app.asana.com/0/1207576726792593/1208782836187274/f

File: articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md (hunk 2 above)
Comment (naomi-lgbt) [reply articles/en/community/deepgram-speech-to-text-aws-lambda-websockets-1304605478935859208.md]: https://app.asana.com/0/1207576726792593/1208782836187274/f

**Pull request merged**